### PR TITLE
Provide a tools to manage the vert.x stack.

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

**Comment by cescoffier** [comment]
@vietj @pmlopes @purplefox ready for review.

File: stack-manager/pom.xml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.vertx</groupId>
+    <artifactId>vertx-stack</artifactId>
+    <version>3.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>stack-manager</artifactId>
```
Comment (vietj) [inline stack-manager/pom.xml]: should be named vertx-stack-manager

**Comment by vietj** [comment]
looks good beside the YAML format we should agree on to use.

**Comment by cescoffier** [comment]
@vietj @purplefox @pmlopes - I've changed the descriptor format to json.

**Comment by pmlopes** [comment]
:+1:

**Pull request merged**